### PR TITLE
feat: add kubernetes scrapper

### PR DESCRIPTION
**Pull request by Kaitou786**

### CODE REVIEW

File: scrapers/kubernetes/kubernetes.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+		for _, obj := range objs {
+			createdAt := obj.GetCreationTimestamp().Time
+			results = append(results, v1.ScrapeResult{
+				BaseScraper: config.BaseScraper,
+				Name:        obj.GetName(),
+				Namespace:   obj.GetNamespace(),
+				Type:        obj.GetKind(),
+				CreatedAt:   &createdAt,
+				Config:      *obj,
```
Comment (Kaitou786) [inline scrapers/kubernetes/kubernetes.go:30]: should we be persisting the entire obj or just the obj.Spec?

cc @moshloop / @Kaitou786

Comment (moshloop) [reply scrapers/kubernetes/kubernetes.go:30]: I think the entire object

File: fixtures/kubernetes.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+kubernetes:
+  -
```
Comment (moshloop) [inline fixtures/kubernetes.yaml:2]:
```suggestion
  - exclusions:
       - Secret
       - ReplicaSet
       - endpoints.discovery.k8s.io
       - endpointslice.discovery.k8s.io
       - lease.coordination.k8s.io
       - podmetrics.metrics.k8s.io
       - nodemetrics.metrics.k8s.io
       - customresourcedefinition
       - controllerrevision
       - certificaterequest
       - order.acme.cert-manager.io
       
       
```

File: api/v1/kubernetes.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package v1
+
+import "github.com/flanksource/kommons"
+
+type Kubernetes struct {
+	BaseScraper     `json:",inline"`
+	Namespace       string          `json:"namespace,omitempty"`
+	UseCache        bool            `json:"useCache,omitempty"`
```
Comment (yashmehrotra) [inline api/v1/kubernetes.go:8]: what does this field do?

Comment (Kaitou786) [reply api/v1/kubernetes.go:8]: If set to false will invalidate the Cache for the client... so the objects are fetched every time.
See: https://github.com/flanksource/ketall/blob/master/client/client.go#L80

**Pull request merged**